src/classes/UTIL_AbstractCommon_BATCH.cls

```diff
@@ -262,16 +262,16 @@ public abstract inherited sharing class UTIL_AbstractCommon_BATCH implements Dat
     @testVisible private String getQueryNonLDVWhereClause() {
```

Comment:
I'm not sure why I didn't notice this before, but `UTIL_AbstractCommon_BATCH` doesn't really say anything about this class.  It's not a general common abstract class for every single batch job. Instead it's explicitly for unique situations where batch chunking is required. The class isn't global so the name can be changed. I think we should definitely considering renaming the class to something like `UTIL_AbstractChunking_BATCH`.

Reply:
It could be, and does provide other functionality, including the ability to provide an additional where clause, validation, etc.  My thought was we might extend/add additional functionality as we find other things we wanted to do across batches.  Even if you aren't running in an LDV mode, there are things it will do, like let you specify a subquery, and have it automatically retrieve that within each batch (instead of the batch needing to explicitly do that).

Not opposed...again I was just thinking that a nice idea for our batches is to extend from this, and allow them to support running both ways (without additional work).  

We can talk more about it tomorrow.

Reply:
Also, this will mean refactoring in 2 additional PRs in Cumulus, plus replicating to Abacus.  If it is important/you feel really strongly I will do it (after we determine whether there is any value for non chunking circumstances), but you will need to twist my arm a bit on this one, as it is a lot of pain...and I really did intend this to be common for both ldv and non ldv concerns-- and that, it should run in non ldv mode unless they record size indicates it needs to run in ldv mode (so we could worry about that less in general) and build with this for new work so even if it eventually grows to that size we do not need to worry about it because it will already be handled.